[2023-08-02 10:44:46,097] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T10:44:44.086027+00:00 [queued]>
[2023-08-02 10:44:46,111] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T10:44:44.086027+00:00 [queued]>
[2023-08-02 10:44:46,111] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-02 10:44:46,111] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-02 10:44:46,111] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-02 10:44:46,187] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_weather_data> on 2023-08-02 10:44:44.086027+00:00
[2023-08-02 10:44:46,194] {standard_task_runner.py:52} INFO - Started process 1813 to run task
[2023-08-02 10:44:46,197] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'S3_to_BigQuery', 'load_weather_data', 'manual__2023-08-02T10:44:44.086027+00:00', '--job-id', '373', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpjv0kf155', '--error-file', '/tmp/tmponsaeuve']
[2023-08-02 10:44:46,198] {standard_task_runner.py:80} INFO - Job 373: Subtask load_weather_data
[2023-08-02 10:44:46,307] {task_command.py:370} INFO - Running <TaskInstance: S3_to_BigQuery.load_weather_data manual__2023-08-02T10:44:44.086027+00:00 [running]> on host f57ce474b9bb
[2023-08-02 10:44:46,571] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=S3_to_BigQuery
AIRFLOW_CTX_TASK_ID=load_weather_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-02T10:44:44.086027+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-02T10:44:44.086027+00:00
[2023-08-02 10:44:47,764] {logging_mixin.py:115} INFO - 
✅ Push to xcom ! ✅
[2023-08-02 10:44:47,765] {python.py:173} INFO - Done. Returned value was: None
[2023-08-02 10:44:47,784] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=S3_to_BigQuery, task_id=load_weather_data, execution_date=20230802T104444, start_date=20230802T104446, end_date=20230802T104447
[2023-08-02 10:44:47,894] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-02 10:44:47,945] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
